test/integration/policy_comp_operator_test.go

```diff
@@ -175,8 +175,7 @@ func complianceScanTest(scanPolicyName string, scanPolicyUrl string, scanName st
 }
 
 var _ = Describe("RHACM4K-2222 GRC: [P1][Sev1][policy-grc] Test compliance operator and scan", Label("policy-collection", "stable", "BVT"), func() {
-	const policyCollectStableURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/stable"
-	const compPolicyURL = policyCollectStableURL + "/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml"
+	const compPolicyURL = policyCollectCAURL + "policy-compliance-operator-install.yaml"
```

Comment:
Should this file also be moved to folder `CA-Security-Assessment-and-Authorization`?

Reply:
I wasn't 100% sure since it's a larger test that includes some CM policies, but it would indeed make sense to move it over--thanks for pointing it out! I'll create some dummy files in the CM folder to direct people over to this file if they're looking for the E8/CIS tests.